schematics/ng-add/schema.ts

```diff
@@ -0,0 +1,6 @@
+export interface Schema {
+  /**
+   * Name of the project where ng-bootstrap should be installed
```

Comment:
Same comment as a previous one (sounds like a lot of 💄)
I think we should be consistent and replace ~~`ng-bootstrap`~~ with `@ng-bootstrap/ng-bootstrap`